storage/dataflux/worksteal.go

```diff
@@ -16,9 +16,20 @@ package dataflux
 
 import (
 	"context"
+	"fmt"
 	"sync"
+	"time"
 
 	"cloud.google.com/go/storage"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	// defaultAlphabet used to initiliaze rangesplitter.
+	defaultAlphabet = "ab"
```

Comment:
Curious what the reasoning behind this default is

Reply:
In rangesplitter we dynamically add more alphabets so object names are not limited to english alphabet. 

For N-base to 10-base conversion there should be at least two default characters for rangesplitter `splitRange` method.
Example - If `defaultAlphabet` is `"a"` , and for `splitRange (start="a",  end="", numSplits=2)` , here end is pointing to the last name after `"a"`.  While calculating 10-base integer value, it will keep appending `"a"` to `start` and `end`.

Reply:
Yes, we'll need at least two characters, could be any character.

Reply:
Please add that in a comment here. Thanks!